Formula/elasticsearch-oss.rb

```diff
@@ -1,21 +1,30 @@
 class ElasticsearchOss < Formula
   desc "Distributed search & analytics engine"
   homepage "https://www.elastic.co/products/elasticsearch"
-  url "https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-7.6.0-darwin-x86_64.tar.gz?tap=elastic/homebrew-tap"
+  if OS.mac?
+    url "https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-7.6.0-darwin-x86_64.tar.gz?tap=elastic/homebrew-tap"
+    sha256 "8414c0ca3feb48a459cb41d555787cb7dd319fdcca25b5f190dd8e4f28ba3e60"
+  else
+    url "https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-7.6.0-no-jdk-linux-x86_64.tar.gz?tap=elastic/homebrew-tap"
```

Comment:
I have the same comments here as the full distribution, let's use the bundled JDK.